src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_container_legal_hold.py

```diff
@@ -11,10 +11,8 @@
 class StorageLegalHold(ScenarioTest):
     @AllowLargeResponse()
     @ResourceGroupPreparer()
-    def test_legal_hold(self, resource_group):
-        storage_account = self.create_random_name('clistorage', 20)
-        self.cmd('storage account create -g {} -n {} --kind StorageV2'.format(
-            resource_group, storage_account))
+    @StorageAccountPreparer(name_prefix='clistorage', length=20, kind='StorageV2')
```

Comment:
Based on my understanding, this will cause the storage account creation request not to be recorded. Is this what we want, given this is storage account's test?

Reply:
This test is for container so the storage account request is just a preparer~

Reply:
Thanks for the explanation.